lib/index.js

```diff
@@ -47,6 +48,20 @@ function register (server, options, next) {
     });
   };
 
+  var orgsEdit = function (id, orgObj, cb) {
+    return query(queries.orgsEdit(id, orgObj), pool,  function (err, res) { //eslint-disable-line
+      if (err) {
+        return cb(err);
+      }
+      // if there were no rows affected, the organisation id does not exist.
+      if (res.rowCount === 0) {
+        return cb(Boom.notFound('That organisation does not exist'));
```

Comment:
We should return null for the response instead of living the value undefined: 
```
return cb(Boom.notFound('That organisation does not exist', null));
```

By defining all the undefined value to null we will be then able to test this value with ```response === null```, it's also a recommendation to use null instead of undefined